src/main/java/dev/jbang/cli/Run.java

```diff
@@ -76,6 +76,12 @@ boolean debug() {
 			"-n", "--native" }, description = "Build via native-image and run", defaultValue = "false")
 	boolean nativeImage;
 
+	@CommandLine.Option(names = { "--deps" }, description = "Add additional dependencies.")
+	List<String> dependencies;
+
+	@CommandLine.Option(names = { "--cp" }, description = "Add class path entries.")
```

Comment:
```suggestion
	@CommandLine.Option(names = { "--class-path", "-classpath", "-cp",  }, description = "Add class path entries.")
```

I haven't tested if picocli actually support the single dash for long names; so please verify - but the idea here is to have the same names as standard `javac` (related to #148)

Reply:
It doesn't:

> Picocli supports POSIX clustered short options: one or more single-character options without option-arguments, followed by at most one option with an option-argument, can be grouped behind one '-' delimiter.

I also think jbang's interface is way to different already to be a drop-in replacement. So having `--cp` instead of `-cp` doesn't feel too bad IMO.

If that's really what you want I think it would be better to come up with a special command (`jbang java`?) that does completely customized parsing and copies the `java` command as much as possible.

Reply:
if not easily feasible to get `-cp` then its fine; close enough is fine :)